app/components/Navbar/Navbar.module.css

```diff
@@ -0,0 +1,23 @@
+/* Navbar Container */
+.navbar {
+    position: absolute;
+    width: 1120px;
+    height: 28px;
+    left: 160px;
+    top: 48px;
+  
```

Comment:
Also, please add CSS for responsivity. The page and navbar should look good on several different screen widths. Research and use common breakpoints. The navbar should stay centered on the screen at all times, and then the links should be consolidated into a hamburger menu when there's not enough space to display all of them in a row.